docs/Matter_TH_User_Guide/Matter_TH_User_Guide.adoc

```diff
@@ -49,8 +49,9 @@ endif::[]
 | Matter 1.1       | v2.8.1             | https://drive.google.com/file/d/15fU3L7QE-MNBslf53A_6sFgn1Wq0Pvqd/view?usp=drive_link[link] | https://groups.csa-iot.org/wg/members-all/document/folder/2730[Causeway Link] | 5a21d17      |
 | Matter 1.2       | TH-Fall2023        | https://drive.google.com/file/d/1WTjhc7xbYt18RvpABU3_r47uqOLd7NN1/view?usp=drive_link[link] | https://groups.csa-iot.org/wg/members-all/document/folder/3045[Causeway Link] | 19771ed      | 
 | Matter 1.3       | v2.10+spring2024   | N/A                                                                                         | https://groups.csa-iot.org/wg/members-all/document/folder/3661[Causeway Link] | 11f94c3      | To install, use the tag "v2.10+spring2024" in the instructions of <<fresh_install>>
-| Matter 1.3       | v2.10.1+spring2024 | N/A                                                                                         | https://groups.csa-iot.org/wg/matter-csg/document/folder/3314[Causeway Link]  | fb2c0e5      | To install, use the tag "v2.10.1+spring2024" in the instructions of <<fresh_install>>
-| Matter 1.4       | v2.11+fall2024 | N/A                                                                                         | https://groups.csa-iot.org/wg/members-all/document/folder/4120[Causeway Link]  | f2e5de7      | To install, use the tag "v2.11+fall2024" in the instructions of <<fresh_install>>
+| Matter 1.3       | v2.10.1+spring2024 | N/A                                                                                         | https://groups.csa-iot.org/wg/members-all/document/folder/3314[Causeway Link] | fb2c0e5      | To install, use the tag "v2.10.1+spring2024" in the instructions of <<fresh_install>>
+| Matter 1.3       | v2.10.2+spring2024 | N/A                                                                                         | https://groups.csa-iot.org/wg/members-all/document/folder/3314[Causeway Link] | 50bcad1      | To install, use the tag "v2.10.2+spring2024" in the instructions of <<fresh_install>>
```

Comment:
Is Matter 1.3 duplicated?

Reply:
`v2.10.1+spring2024` and  `v2.10.2+spring2024`

Reply:
`v2.10+spring2024`  too

Reply:
Oh, I see